### PR TITLE
chore PEP8standards

### PR DESCRIPTION
**Pull request by magicmarie**
What does this PR do?
This commit fixes the project to PEP8 standards

Description
The project python files were not up to PEP8 standards.

### CODE REVIEW

File: hc/accounts/tests/test_check_token.py [hunk 1]
```diff
@@ -25,15 +25,26 @@ def test_it_redirects(self):
         self.assertEqual(self.profile.token, "")
 
     def test_redirects_already_logged_in(self):
-        """test that a logged in user redirects when they log in again""" 
-        #Login and test it redirects already logged in
-        self.client.post(reverse('hc-check-token', args = ["alice", "secret-token"]))
-        re = self.client.post(reverse('hc-check-token', args = ["alice", "secret-token"]))
+        """test that a logged in user redirects when they log in again"""
+        # Login and test it redirects already logged in
```
Comment (JaymesKat) [inline hc/accounts/tests/test_check_token.py]: Given that you have already included a docstring on the previous line, I think adding a single line comment would be a duplication.

Comment (magicmarie) [reply hc/accounts/tests/test_check_token.py]: pep8 changed line 29 to that. must have been ###

**Comment by EdwinKato** [comment]
@magicmarie rectify the `flake8` dependency issue with `pycodestyle`. Running flake8 on `hc` causes an error

**Comment by EdwinKato** [comment]
There are also still a number of areas in the code that are not following pep8. See image below

screen shot 2018-07-08 at 13 01 18

**Comment by magicmarie** [comment]
@EdwinKato more clarity on ``` rectify the flake8 dependency issue with pycodestyle. Running flake8 on hc causes an error```. I ran flake8 and there were no errors except the flake8 complaints.

**Comment by magicmarie** [comment]
@EdwinKato  i worked on the pep8 errors. The ones left are ```variable assigned but not used```. They are important and can't be deleted.

**Comment by EdwinKato** [comment]
@magicmarie Rectify the errors that appear when you try running the application or just running tests. The `variable assigned but not used` errors can also be done away with. Just modify the particular instances to call the method without having to assign the result to any variable

**Review by EdwinKato — APPROVED** [review]

**Pull request merged**